addons/account/models/account_move.py

```diff
@@ -1548,6 +1548,10 @@ def _check_duplicate_supplier_reference(self):
         ''', [tuple(moves.ids)])
         duplicated_moves = self.browse([r[0] for r in self._cr.fetchall()])
         if duplicated_moves:
+            posted_move = duplicated_moves.filtered(lambda r: r.name != '/')
+            amount_total = sum(posted_move.mapped('amount_total')) + self.amount_total
+            if amount_total <= duplicated_moves.reversed_entry_id.amount_total:
+                return
```

Comment:
I'm not sure this is the best place to do that (although I can't think of a better one), but if I were to do it here, I would probably do something like this :
```suggestion
            # If duplicated_moves are reversal moves, we do not raise a ValidationError.
            if duplicated_moves.reversed_entry_id and len(duplicated_moves.reversed_entry_id) == 1:
                posted_reversal_moves = duplicated_moves.filtered(lambda move: move.state == 'posted')
                amount_total_reversal_moves = sum(posted_reversal_moves.mapped('amount_total')) + self.amount_total
                amount_total_reversed_entry = duplicated_moves.reversed_entry_id.amount_total
                if amount_total_reversal_moves <= amount_total_reversed_entry:
                    return
                raise UserError(
                    _('The total amount of the reversal moves (%s) exceeds the amount of the reversed entry (%s).') % (
                        formatLang(self.env, amount_total_reversal_moves, currency_obj=self.currency_id),
                        formatLang(self.env, amount_total_reversed_entry, currency_obj=self.currency_id)
                    )
                )
```
What do you think @smetl ?